recipes/spfft/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "spfft" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/eth-cscs/SpFFT/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 541f6d92993655b68404cfea60d84ce0a7644eed49dff7e7437db818127700f9
+
+build:
+  number: 0
+  skip: True  # [ win ]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - cmake >=3.11.0
+  host:
+    - {{ mpi }}
+    - fftw
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+  run:
+    - {{ mpi }}
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib64/libspfft${SHLIB_EXT}" || test -f "${PREFIX}/lib/libspfft${SHLIB_EXT}"
```

Libraries should always be in `lib` on conda-forge.
```suggestion
    - test -f "${PREFIX}/lib/libspfft${SHLIB_EXT}"
```